src/coreclr/jit/CMakeLists.txt

```diff
@@ -41,6 +41,9 @@ function(create_standalone_jit)
   elseif(TARGETDETAILS_ARCH STREQUAL "s390x")
     set(JIT_ARCH_SOURCES ${JIT_S390X_SOURCES})
     set(JIT_ARCH_HEADERS ${JIT_S390X_HEADERS})
+  elseif(TARGETDETAILS_ARCH STREQUAL "loongarch64")
```

Comment:
Changes in this file IMO belongs to https://github.com/dotnet/runtime/pull/62893 . All files from that CMake file are seems to be missing from all PRs which you are sent so far. Maybe these files belongs to #62893 too.

Reply:
Thanks very much.

At the beginning, I thought if the last patch, that is this PR, was added the whole will be work, so I move all the building configures at this PR.

I will move the related files to the corresponding PR.